services/state/primitives.proto

```diff
@@ -24,8 +24,8 @@ package proto;
  * type instead.
  */
 
-option java_package = "com.hederahashgraph.api.proto.java.primitive";
-// <<<pbj.java_package = "com.hedera.hapi.node.state.primitive">>> This comment is special code for setting PBJ Compiler java package
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.state.primitives">>> This comment is special code for setting PBJ Compiler java package
```

Comment:
This should be singular, to match the rest of the codebase which doesn't really use plural for package names.

Reply:
I agree with you, but changing this is outside the scope of this PR. It was changed in PR #302 and is only showing up here because I merged that change in.

Reply:
It should not have changed in that PR, then.